well-sensor-esp32.ino

```diff
@@ -83,7 +85,16 @@ void loop() {
   mqttClient.loop();
   
   dance(100);
+
+  if (current_sample < SIZE) {
+    
+    data[current_sample++] = getNumber();
+    return;
+  }
   
```

Comment:
Pls introduce function for this in init-data.h

Reply:
This routine can`t be in init-data.h, cause we use function from data-median.h(getDiscreteNumber()).
Currently this part of code is extracted to function and moved to data-median.h file.

Reply:
why?
keep getDiscreteNumber call out of the function